backend/aks/deploy.sh

```diff
@@ -14,6 +14,8 @@ if ! az account show; then
        --username "${AZURE_APP_ID}" \
        --password "${AZURE_PASSWORD}" \
        --tenant "${AZURE_TENANT_ID}"
+
+    az account set --subscription="${AZURE_SUBSCRIPTION_ID}"
```

Comment:
Nitpicking: I would then change the print of `info "Missing azure credentials, running az login…"` to `login into the account` or so.

Reply:
done.